mcp/README.md

```
@@ -219,6 +220,59 @@ types passwords.
 `bash` · `job_status` · `job_stop` · `read_file` · `write_file` · `edit_file` ·
 `list_directory` · `glob` · `grep` · `notebook_edit` · `web_fetch` · `web_search`
 
+### Coding agent (both editions, only while the owner's switch is on)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)